gemfiles/sass_3_4.gemfile

```diff
@@ -16,7 +18,6 @@ gem "pg"
 gem "rails", "4.2.2"
 gem "recipient_interceptor"
 gem "refills"
-gem "sass-rails", "~> 5.0"
```

Comment:
yep - I think this appraisal gemfile was out of date with the one on master. It should be caught up as of #224.

Rebase if you want to, otherwise it'll come out in the wash.